clients/fides-js/src/components/ConsentButtons.tsx

```diff
@@ -52,7 +52,9 @@ export const ConsentButtons = ({
     {includePrivacyPolicy && <PrivacyPolicyLink i18n={i18n} />}
     <div
       className={
-        isInModal ? "fides-modal-button-group" : "fides-banner-button-group"
+        isInModal
+          ? "fides-modal-button-group"
```

Comment:
```suggestion
          ? "fides-modal-button-group fides-modal-primary-actions"
```
Just for consistency...